src/components/common/YourCompany/CorrectName/CorrectNameRequest.vue

```diff
@@ -214,33 +214,33 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
             'OK'
           )
         } else {
-          this.parseNameRequest(nr)
+          this.storeNameRequest(nr)
           // Set our entity type, if it's a conversion request
           if (nr.request_action_cd === NrRequestActionCodes.CONVERSION) {
-            this.setEntityType(nr.legalType)
+            this.setEntityType(nr.legalType as unknown as CorpTypeCd)
             this.setEntityTypeChangedByName(true)
           }
           this.emitIsSaved(true)
         }
       } catch {
-        // "validateNameRequest" handles its own errors
+        // "fetchValidateNameRequest" handles its own errors
         // Inform parent process is complete
         this.emitIsSaved()
       }
     }
   }
 
   /* Checks name request type or if it's an invalid conversion name request. */
-  isNameRequestInvalid (nr: NrResponseIF): boolean {
-    const isNameEntityTypeDifferent = this.getEntityType !== nr.legalType
+  isNameRequestInvalid (nr: NameRequestIF): boolean {
+    const isNameEntityTypeDifferent = (this.getEntityType !== nr.legalType as unknown as CorpTypeCd)
     const entityTypeOptions = this.getResource?.changeData?.entityTypeOptions
     const isValidConversionNameRequest = nr.request_action_cd === NrRequestActionCodes.CONVERSION &&
       entityTypeOptions?.some(options => options.value === nr.legalType)
     return (isNameEntityTypeDifferent && !isValidConversionNameRequest)
   }
 
   /* Generate content of error depending on name request type. */
-  nameRequestErrorText (nr: NrResponseIF): string {
+  nameRequestErrorText (nr: NameRequestIF): string {
```

Comment:
This is a noteworthy change -- instead of using the NR in its Legal API format, we use the same shared IF everywhere now.